tests/unit_tests/test_material.py

```diff
@@ -34,6 +34,22 @@ def test_elements():
     with pytest.raises(ValueError):
         m.add_element('Pu', 1.0, enrichment=3.0)
 
+def test_elements_by_name():
+    """Test adding elements by name"""
+    m = openmc.Material()
+    m.add_element('woLfrAm',1.0)
```

Comment:
```suggestion
    m.add_element('woLfrAm', 1.0)
```
I had no idea! Now "W" makes sense :)

Reply:
I think I've spent too much time reading fun element facts by now! I was surprised to discover it's apparently a common translation across parts of Europe and is still sometimes used here in the UK, too.